ProcessMaker/processmaker
# FOUR-6733: Import/Export | Update Export Process Main Page

# Issue
Ticket: [FOUR-6733](https://processmaker.atlassian.net/browse/FOUR-6733)

The main process export page `processes/{processId}/export` needs to be updated to match the new design specified by the Import/Export [PRD](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/2719481999/PRD+PM4+Export+Import#New-Designer-View.).

# Solution
Update Export Process main page to include flows for Basic and Custom exports.

# Current UI
<img width="1432" alt="Screen Shot 2022-09-27 at 11 10 20 AM" src="https://user-images.githubusercontent.com/73713665/192565030-96b6f9f3-396f-4350-9b7c-4125607ae067.png">

# Updated UI
<img width="1432" alt="Screen Shot 2022-09-27 at 11 04 02 AM" src="https://user-images.githubusercontent.com/73713665/192563847-2aa158db-262c-4e38-8d0b-3833591349dd.png">

# How to Test
1. Update core branch to `bugfix/FOUR-6733`.
2. Go to ProcessMaker -> Designer -> Export a Process
3. Export Page styling should match [PRD](https://processmaker.atlassian.net/wiki/spaces/PM4/pages/2719481999/PRD+PM4+Export+Import#New-Designer-View.).
4. Basic Export should be selected by default.
5. Selecting Basic Export should take you to a Set Password Modal.
6. **Note:** Selecting Custom Export should take you to an Advanced Export Process screen. Currently, it takes you to the Set Password Modal because the Advanced Export Process screen has not yet been built out.

# Related Tickets & Packages
- [FOUR-6570](https://processmaker.atlassian.net/browse/FOUR-6570) | Import/Export Epic

# Code Review Checklist
